block-layer/codecs/dag-pb.md

```diff
@@ -36,15 +36,48 @@ message PBNode {
 The objects link names are specified in the 'Name' field of the PBLink object.
 All link names in an object must either be omitted or unique within the object.
 
-## Pathing
+## Logical Format
 
-The pathing is currently different between implementations. Please see [issue #55] for more information about the harmonization effort. This section describes the current implementations as of September 2019.
+When we handle DagPB content at the Data Model level, we treat these objects as maps.
+The map keys correspond to the lowercase field names.
```

Comment:
Is this lower-casing true in Go? It's not in [JS](https://github.com/ipld/js-ipld-dag-pb), we have `node.Data` and `node.Links`, then `link.Name`, `link.Tsize` and `link.Hash`.

I wouldn't even say we treat it as a map if I were being precise about the way it's handled in JS, but maybe the rough idea is good enough to state that way.

Reply:
Uuf, you're right: https://github.com/ipld/go-ipld-prime-proto/blob/e32bd156a1e52653dbdb8b1696d74205a20f220c/pb_node_gen.go#L599-L619 it's currently implemented in uppercase in the new stuff as well.

I'm not sure if I wanna fix our doc here or if I want to change the implementation.  This capitalization is distinctly weird.

(It's *definitely* and absolute [a map](https://github.com/ipld/go-ipld-prime-proto/blob/e32bd156a1e52653dbdb8b1696d74205a20f220c/pb_node_gen.go#L1356-L1358) in go, though.)

Reply:
I think also in the Data Model it should be the capitalized names, so that it corresponds to the Protobuf names (1:1 mapping). I think a rename is fine here, but we can also use the capitalized names directly (I kind of like the rename thing, to show the power of IPLD Schemas).

Reply:
we should just make the schema read exactly like the Data Model because when you sit down to write selectors for this stuff it’s going to be against the Data Model anyway. it’s going to be super painful if we have to get everyone to adopt a schema rename in their selector engine in order to handle the queries being written against the renamed keys.

Reply:
I've unwound the two mentions of lowercase, and moved the schema to just use uppercase. ptal